include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -682,14 +693,29 @@ struct __result_and_scratch_storage : __result_and_scratch_storage_base
         }
     }
 
-    template <typename _Event>
-    _T
-    __wait_and_get_value(_Event&& __event, size_t idx = 0) const
+    template<typename _Type>
+    std::size_t
+    __fill_data(std::pair<_Type, _Type>&& __p, std::size_t* __p_buf) const
+    {
+        __p_buf[0] = __p.first;
+        __p_buf[1] = __p.second;
+        return 2;
+    }
+
+    template<typename _Args>
+    std::size_t
+    __fill_data(_Args&&...) const
+    {
+        assert(!"Unsupported return type");
+        return 0;
+    }
+
+    virtual std::size_t __get_data(sycl::event __event, std::size_t* __p_buf) const override
```

Comment:
Technically this declaration is correct.
But for compatibility with the other code, as I seen, `virtual` aren't used together with `override` in our code.

Reply:
You mean "for consistency", "not compatibility"...
1) Basically, 99% oneDPL code doesn't use even "virtual", because 99% of code is general template code, which provides static polymorphism.
2) `override` is available since C++11 , we can use `override` keyword. It gives more compile time coding control.
Not using it before  - it was a "gap".

Reply:
So if it is a "gap" - could you please write an issues?

Reply:
https://github.com/uxlfoundation/oneDPL/issues/2005